dumper/app/src/main/resources/oracle-stats/native/db-objects-synonym-public.sql

```diff
@@ -0,0 +1,50 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  B.con_id "ConId",
+  B.editionable "Editionable",
```

Comment:
Can you add `ObjectType` and `Owner` columns to make schema compatible with `db-objects-no-join`?

Reply:
Done, I added the columns